war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -2292,7 +2291,7 @@ function updateBuildHistory(ajaxUrl,nBuild) {
         }
 
         function loadPage(params, focusOnSearch) {
-            var searchString = pageSearchInput.value;
+            const searchString = pageSearchInput.value;
```

Comment:
Cannot use `const` on the `hudson-behavior.js` file as it's not processed through webpack. These should be `var`s

Reply:
Done.